staging/src/k8s.io/apiserver/pkg/util/shufflesharding/shufflesharding_test.go

```diff
@@ -277,7 +276,7 @@ func TestDealer_DealIntoHand(t *testing.T) {
 		},
 		{
 			"size: 6 cap: 6 slice",
-			make([]int, 6, 6),
+			make([]int, 6),
```

Comment:
@lavalamp  the previous comment was around this line of code...   the comment was:
> hm since this file is making assertions about the slice I think it'd be good to keep the cap set deliberately. Since without this it's not obvious that you won't get a larger cap than len.

The comment / question is:  `make` explicit sets the cap to the length unless specified.   Is this sufficient?  or are you suggesting that adding the addition of an explicit cap makes it more readable?   The explicit cap (which is redundant to the default) is considered a staticcheck violation... do you suggest an inline ignore?    

Thank you sir!

Reply:
I guess it is clear from the string what the author believes this is testing, that is probably good enough. Thanks.